tinyagentos/tools/cluster_tools.py

```
@@ -0,0 +1,107 @@
+        return out
+    try:
+        for be in catalog.backends_with_capability("image-generation"):
+            out.append({
+                "name": be.name,
+                "type": be.type,
+                "tier": _TIER.get(be.type, "unknown"),
+                "loaded": getattr(be, "lifecycle_state", "running") == "running",
+                "models": [m.get("id") or m.get("name") for m in (be.models or [])][:10],
+            })
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)